src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * The class that deals with loading and saving tasks in the file.
+ */
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * The method which loads the file.
+     * @return The list of tasks that were stored in the file.
+     */
+    public ArrayList<Task> load() {
+        File tasks = new File(filePath);
+        tasks.getParentFile().mkdirs();
+        ArrayList<Task> taskList = new ArrayList<>();
+        try {
+            if (!tasks.createNewFile()) {
+                try {
+                    BufferedReader reader = new BufferedReader(new FileReader(tasks.getPath()));
+                    label:
+                    for (String line; (line = reader.readLine()) != null; ) {
+                        String[] taskInfo = line.split("[*]", 4);
+
+                        switch (taskInfo[0]) {
+                            case "T":
```

Comment:
According to [Java Coding Standards](https://se-education.org/guides/conventions/java/intermediate.html), there should be no indentation for `case` clauses.